generate-clang-config.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
TIL: you don't even need this, it's totally fine just to create `<triple>-clang` and the target will automatically be inferred from the symlink name.